hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -3441,11 +3442,13 @@ func (r *HostedClusterReconciler) reconcileMachineConfigServer(ctx context.Conte
 	}
 
 	// Reconcile service
-	mcsService := ignitionserver.MCSService(controlPlaneNamespace.Name)
-	if _, err := createOrUpdate(ctx, r.Client, mcsService, func() error {
-		return reconcileMachineConfigServerService(mcsService)
-	}); err != nil {
-		return fmt.Errorf("failed to reconcile machine config server service: %w", err)
+	if _, exists := hcluster.Annotations[hyperv1.DisableMonitoringServices]; !exists {
```

Comment:
can you please just remove this service? This is not used anymore.
cc @enxebre

Reply:
Thank you for the review Cesar. Just to make sure we are on the same page, are you referring to removing the whole `if _, exists := hcluster.Annotations[hyperv1.DisableMonitoringServices]; !exists {` if-statement section?

Reply:
yes